### PR TITLE
Fix error code for invalid zip file content

### PR DESCRIPTION
**Pull request by rkondratowicz**
https://tools.hmcts.net/jira/browse/BPS-1173

### CODE REVIEW

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/InvalidZipFilesException.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5335041&bid=17463240)

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java [hunk 1]
```diff
@@ -114,7 +115,7 @@ static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSi
 
     static void verifyFileNames(Set<String> fileNames) {
         if (!(fileNames.size() == 2 && fileNames.containsAll(asList(DOCUMENTS_ZIP, SIGNATURE_SIG)))) {
-            throw new DocSignatureFailureException(
+            throw new InvalidZipFilesException(
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java:118]: does it need to be caught specifically somewhere similarly as `DocSignatureFailureException`?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java:118]: good point, I'll check

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/InvalidZipFilesException.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5335041&bid=17463240)

**Pull request closed without merging**